config/initializers/email.rb

```diff
@@ -0,0 +1,7 @@
+if Rails.env.development?
+  ActionMailer::Base.delivery_method = :smtp
+  ActionMailer::Base.smtp_settings = {
+    address: "localhost",
+    port: 1025
+  }
+end
```

Comment:
We moved the code from development.rb to production.rb as that is where it belongs.